src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faMicrophone,
+  faCog,
+  faChevronLeft,
+} from '@fortawesome/free-solid-svg-icons';
+import { useSelector } from 'react-redux';
+import classes from './Navbar.module.css';
+
+function Navbar() {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { continent } = useSelector((state) => state.continentReducer);
+
+  return (
+    <nav className={classes.nav}>
+      {location.pathname === '/' && (
+        <>
+          <p>
+            <b>The World</b>
+          </p>
+          <p className={classes.shrink}>Continents COVID Cases</p>
```

Comment:
- [ ] When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name. Please don't forget to add a search bar. 😉

Reply:
Done

Reply:
Done